tests/run.py

```diff
@@ -22,14 +29,30 @@
                         fight_style=fight_style, profile=path)
         tests.append(grp)
         for talents in talent_combinations(klass):
-            Test(talents, group=grp, args=[ ('talents', talents) ])
+            Test(talents, group=grp, args=[('talents', talents)])
 
     for fight_style in FIGHT_STYLES:
         grp = TestGroup('{}/{}/covenants'.format(profile, fight_style),
                         fight_style=fight_style, profile=path)
         tests.append(grp)
         for covenant in COVENANTS:
             Test(covenant, group=grp,
-                 args=[ ('covenant', covenant.lower()), ('level', 60) ])
+                 args=[('covenant', covenant.lower()), ('level', 60)])
 
-sys.exit( run(tests) )
+    if not args.no_trinkets:
+        from simc_support.game_data.WowSpec import get_wow_spec_from_combined_simc_name
+        from simc_support.game_data.Trinket import get_trinkets_for_spec
+
+        trinkets = get_trinkets_for_spec(
+            get_wow_spec_from_combined_simc_name(klass))
+        fight_style = args.trinkets_fight_style
+        grp = TestGroup('{}/{}/trinkets'.format(profile, fight_style),
+                        fight_style=fight_style, profile=path)
+        tests.append(grp)
+        for trinket in trinkets:
+            Test('{} ({})'.format(trinket.name, trinket.item_id), group=grp, args=[
+                    ('trinket1', '{},id={},ilevel={}'.format(trinket.name, trinket.item_id, trinket.min_itemlevel))])
+else:
+    print("No profile found for {}".format(klass))
```

Comment:
This seems to be printed out for all specs even if there actually is a profile.
I believe an `else` branch bound to a loop gets executed if there is no `break` in the loop. Or smth like that.